src/borg/testsuite/platform.py

```diff
@@ -44,6 +46,25 @@ def user_exists(username):
         return False
 
 
+def bad_acl_implementation():
+    global _bad_acl_implementation
```

Comment:
could this have a non-negated / non-negative name?
like are_acls_supported/working?